HW 1/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Homework</title>
+</head>
+<body>
+
+    <button id="myBtn"> Try it </button>
+    
+    <button class="myTogBtn"> Toggle Button </button>
+
+    <script>
+

```

Comment:
No need for this space line here.

Reply:
Thank you very much for your detailed feedback.
Have a nice weekend teacher